spec/support/simple_work.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Hyrax
+  module Test
+    ##
+    # A generic PCDM Work, with only Hyrax "core" (required) metadata.
+    #
+    # @example building with FactoryBot
+    #   work = FactoryBot.build(:work, :public, title: ['Comet in Moominland'])
```

Comment:
Why not use `:hyrax_work`, but instead using `work` here?

Reply:
good catch. i'll fix this.

Reply:
updated.

Reply:
but i didn't see there is `title` defined in `:hyrax_work` factory: https://github.com/samvera/hyrax/pull/4053/files#diff-82cb20478049021cf266e1d389b3ebea
Did i miss something?

Reply:
fields only need to be defined on factories to provide default/factory values. `#title` is defined on `SimpleWork`, and that is sufficient here.

Reply:
oh, I see. Thanks!